### ISSUE TITLE
A question on loss of `mix_refpoint`

### ISSUE THREAD

**Issue by JunrQ** [Michel-liu/GroupPose#2]
Hello, thank you for your wonderful work, i have a question,

in https://github.com/Michel-liu/GroupPose/blob/9ed9e7ef4e3b247242700691ca1db92e194b5537/models/grouppose/transformer.py#L167C60-L167C95 ,

the mean of keypoint coordinates is concatenated in front, but the keypoints loss is calculated for first 17/14 keypoints, is that a bug?

**Comment by Michel-liu** [comment]
Thank you for your report ❤️! I have checked the code and will fix this issue soon.

**Comment by JunrQ** [comment]
Dear author, I trained the resnet coco and the ap is still 0 after 15 epoches, is that normal?

**Comment by Michel-liu** [comment]
That's a strange result. Can you provide more information, such as the output log and training args you used?
BTW, with the 12 epochs training setting, the AP comes to 56.2 and 60.1 at the 10th and 11th epochs.

**Comment by JunrQ** [comment]
Thank you for your quick reply, previous i trained with 6 gpus and the model failed to converge, but when i retrained with 8 gpus, it seems normal.

**Comment by rrenle** [comment]
> Thank you for your quick reply, previous i trained with 6 gpus and the model failed to converge, but when i retrained with 8 gpus, it seems normal.

Hi, can you reproduce the results of the paper? My result is AP=70.5 which is lower than 72.
